api/src/game.ts

```diff
@@ -49,13 +59,11 @@ export const switchToNextPlayer = (
   actualPlayer: Player,
   players: Array<Player>,
 ): Player => {
-  let nextPlayerIndex = players.indexOf(actualPlayer) + 1;
-
-  if (nextPlayerIndex > players.length - 1) return players[0];
-  return players[nextPlayerIndex];
+  if (actualPlayer.playerNumber === 1) return players[1];
+  return players[0];
 };
 
-export const marbleWonByPlayer = (graph: Graph): number => {
+export const marbleColorExited = (graph: Graph): number => {
```

Comment:
As said before, `marbleColorExited` doesn't explain what this function does. I got a clue by reading the tests. Maybe `getMarbleWonByPlayer`?

Reply:
Ok, i see, this is obvious. corrected